joomla/joomla-cms
# Allows calendar strings override per language

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/20296

### Summary of Changes
Let's add in a media/xx-XX/ language folder a xx-xx.js lang file for calendar on the model of what we already do for other calendar files for fa-IR for example.

### Testing Instructions
Example with en-US which is normally using the en.js file from
`/ROOT/media/system/js/fields/calendar-locales/en.js`

Create the following hierarchy for en-US in the media folder
![screen shot 2018-05-05 at 09 02 17](https://user-images.githubusercontent.com/869724/39660617-39461f48-5043-11e8-8778-159bdc830864.png)

the en-us.js file is a copy of the en-js file where you will modify some values.
Install en-US language and switch to it in backend, edit an article, check the values you modified.

@dgrammatiko @alikon
